plugin/secrets_access_token.go

```diff
@@ -73,8 +74,9 @@ func (b *backend) secretAccessTokenResponse(ctx context.Context, s logical.Stora
 
 	return &logical.Response{
 		Data: map[string]interface{}{
-			"token":      token.AccessToken,
-			"expires_at": token.Expiry,
+			"token":           token.AccessToken,
+			"token_ttl":       time.Now().Sub(token.Expiry),
+			"expires_at_unix": token.Expiry.Unix(),
```

Comment:
"Unix" could be interpreted as unix seconds or unix with microseconds or unix with milliseconds. I would recommend we just say "expires_at_seconds" or "expires_in_seconds"